soccer/gameplay/single_robot_composite_behavior.py

```diff
@@ -10,6 +10,24 @@
 # Note: The method-resolution-order in python dictates that as we have it listed below, SingleRobotBehavior methods take precedence over CompositeBehavior methods
 class SingleRobotCompositeBehavior(single_robot_behavior.SingleRobotBehavior,
                                    composite_behavior.CompositeBehavior):
+    #If the behavior should not be restarted when it is assigned to a different robot, then restart_behavior_on_switch_robot must be false
+    def __init__(self,
+                 continuous=False,
+                 restart_behavior_on_switch_robot=True):
```

Comment:
@jcarn did you decide against the lambda option (and renaming this variable?) I think that maybe some of your commits got dropped when you pushed the new version, I can help you recover them if you flag me down the next time you see me. 

I'm fine with leaving out the lambda for now, but I'd really like this name to be shorter :P